src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -1057,7 +1058,8 @@ private DataRepoResponse<DatasetDataModel> retrieveDatasetDataByIdRaw(
     request.filter(Objects.requireNonNullElse(filter, ""));
     request.sort(Objects.requireNonNullElse(sort, PDAO_ROW_ID_COLUMN));
     request.setDirection(
-        Objects.requireNonNullElse(SqlSortDirection.fromValue(direction), SqlSortDirection.ASC));
+        SqlSortDirectionAscDefault.valueOf(
+            Objects.requireNonNullElse(direction, SqlSortDirection.ASC.name())));
```

Comment:
This works but is there a reason to not change the API to wire through `SqlSortDirectionAscDefault` as the argument type?

That is, change the `direction` parameter of `retrieveDatasetDataByIdRaw()` to be `SqlSortDirectionAscDefault`, and change the methods that pass in `null` to instead pass `SqlSortDirectionAscDefault.ASC`. Once that's done there are only two cases left where an explicit sort order is passed, and those can be changed to pass the enum type instead of a string, for example:
```
                String.valueOf(SqlSortDirection.ASC))
```

to
```
                SqlSortDirectionAscDefault.ASC)
```

Then this line can be
```
    request.setDirection(direction);
```